README.md

```diff
@@ -49,10 +49,13 @@ The Make and CMake build systems create three libraries: libmbedcrypto, libmbedx
 
 You need the following tools to build the library with the provided makefiles:
 
-* GNU Make or a build tool that CMake supports.
+* GNU Make 3.82 or a build tool that CMake supports.
 * A C99 toolchain (compiler, linker, archiver). We actively test with GCC 5.4, Clang 3.8, IAR8 and Visual Studio 2013. More recent versions should work. Slightly older versions may work.
-* Python 3 to generate the test code, and to generate sample programs in the development branch.
+* Python 3.6 to generate the test code, and to generate sample programs in the development branch.
 * Perl to run the tests, and to generate some source files in the development branch.
+* CMake 3.10.2 or later (if using CMake).
+* Microsoft Visual Studio 2013 or later (if using Visual Studio).
```

Comment:
I get that Visual Studio is both a build automation tool and a C compiler, but it looks weird to mention the same product twice.

Reply:
It does, but the previous mention is a more vague "we test with ...", so this is saying something a bit more definitive. If we didn't have the second line someone could interpret the document as saying "this slightly older version of VS should work".

Reply:
We know that VS2010 doesn't work because it completely lacks C99 support. VS2012 may or may not work.